.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Can we keep this config the same as grove core? Having differing styles between repos can be frustrating.

Reply:
@racoleman we can but we do not need nearly as many rules as we do in grove core. I opted for a simpler config for that reason, as well as rules that are more fitting for a node environment IMO

Reply:
Simpler I understand, but we should keep things consistent (e.g. semicolons, as @jessewagstaff pointed out).

Reply:
@racoleman I don't know if I agree. The target here is different and some of the options here are preferred in the node.js environment. I think we should opt for it. Consistency for the sake of consistency doesn't matter, especially when it adds complexity and bringing over potential baggage from other or older styles and practices. It's the end goal that matters. 

This is also why I don't think it's in this package's best interest to be maintained by the grove organization. This is a general open source project were people can contribute outside of grove. Simpler in this case is almost always better.